pallets/identity/src/lib.rs

```diff
@@ -126,6 +126,37 @@ decl_storage! {
         /// All authorizations that an identity/key has given. (Authorizer, auth_id -> authorized)
         pub AuthorizationsGiven: double_map hasher(blake2_256) Signatory, blake2_256(u64) => Signatory;
     }
+    add_extra_genesis {
+        config(identities): Vec<(T::AccountId, IdentityId, IdentityId, u64)>;
+        build(|config: &GenesisConfig<T>| {
+            for &(ref master_account_id, did_issuer, did, ref expiry) in &config.identities {
+                use chrono::prelude::Utc;
+                // Direct storage change for registering the DID and providing the claim
+                let master_key = AccountKey::try_from(master_account_id.encode()).unwrap();
+                assert!(!<DidRecords>::exists(did), "Identity already exist");
+                <MultiPurposeNonce>::mutate(|n| *n += 1_u64);
+                <Module<T>>::link_key_to_did(&master_key, SignatoryType::External, did);
+                let record = DidRecord {
+                    master_key,
+                    ..Default::default()
+                };
+                <DidRecords>::insert(&did, record);
+
+                // Add the claim data for the CustomerDueDiligence type claim
+                let current_timestamp = ((Utc::now()).timestamp() * 1000) as u64;
```

Comment:
Wouldn't `Utc::now()` return the current system time? The current time can vary between systems. Everything here should be deterministic.

Reply:
Yeah, I used an approach used in `pallet-timestamp`